demo/src/routes/menu/CollapsibleSection.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+	import {collapseVerticalTransition} from '@agnos-ui/core/services/transitions/bootstrap';
+	import {createTransition} from '@agnos-ui/core/services/transitions/baseTransitions';
+	import {writable} from '@amadeus-it-group/tansu';
+
+	export let headerText;
+	const paramTransition$ = writable(collapseVerticalTransition);
+	const paramAnimation$ = writable(true);
+	const paramVisible$ = writable(false);
+
+	const {
+		stores: {visible$},
+		api: {toggle},
+		directives: {directive},
+	} = createTransition({
+		props: {
+			animation: paramAnimation$,
+			visible: paramVisible$,
+		},
+	});
+</script>
+
+<div class="mb-2">
+	<button class="btn btn-light btn-lg p-1 d-flex align-items-center btn-header" aria-expanded={$visible$} on:click={() => toggle()}
+		>{headerText}
+		<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 100 100">
```

Comment:
shouldn't we reuse icons of accordion?

Reply:
wanted to do something different from accordion, this feels different and more fun :) FYI didn't use the accordion itself cause the only massive thing that connects these two is a transition and this is a pretty simple case